### DISCUSSION TITLE
DEM raster (veg removed) to Spline/Bezier curve contors to OpenOrienteeringMpaper

### DISCUSSION THREAD

**Discussion by CairnCoder** [OpenOrienteering/mapper#2061] [Questions & Answers]
I am in the following situation:

I have:
laz file of lidar data.
DEM raster (with vegetation removed).

I want:
Nice smooth contors in OpenOrienteeringMpaper

Does anyone have some advice on how to do this. E.g. A particular workflow.

What I have done:
Import the raster in to qgis.
In qgis, resampled the raster with an average of the near cells/pixels, to give a new averaged height for each cell/pixel.
In qgis, generated contors.
In qgis, smoothed these contors.
Exported form qgis.
Imported in to OpenOrienteeringMpaper as the contour line symbol.
**This gives me nice smooth contors but the vectors are actually made of many many points. I am happy with this outcome but the file size is large compared to what it should be, is 4MB instead of 0.8MB .**
I have tried using the vector smoothing button (converts the vector in to an spline/Bezier curve) but to outcome is nowhere near as nice as the point vectors. I.e. It often occurs that, the arms of each anchor are very small thus producing a sharp corner where it should have been smooth.
What am I doing wrong with it?

I don't mind to minor edits to contour lines but I am looking for most of them to be autogenerated.

So my questions are:
Can I produce a spline/Bezier curve file from qgis?
Can I then import this in to OpenOrienteeringMpaper as splines/Bezier curves?
And how do I do this?

Any help would be much apricated.

**Comment by Choreuomai** [comment]
How did you smothe the contours in QGIS? Via Vector->Generalizer (add-on)? There are a number of options there and I ended up using Boyle's forward-looking algorithm (without knowing anything about it), which produced less than 4MB for an ordinary-sized O-map.

**Comment by jmacura** [comment]
Hi,

simplify the contour lines in QGIS should help.
Another button worth a try is the "simplify" tool in OO-Mapper. Simplify the lines before you convert them to curves.

Regards

edit: I missed that the question is 2 years old, still HTH someone.

**Reply by Choreuomai** [reply]
Yes, I also thought it still is an interesting issue. My comment was actually meant as half-answer/half-question:
 I do "smoothe/simplify" the contours in QGIS _by_ using the Generalizer plugin, which in my opinion addresses the issue mentioned in the original post (namely that the file is larger than it should be). So that was the answer part of my comment.
My question is: Do you do anything else (other than using the generalizer plugin) to "smoothe/simplify" the contours?

**Reply by sfroyen** [reply]
I think it's better to smooth the terrain and use the smoothed DEM to generate contours. The smoothing algorithm should adapt and avoid smoothing across, e.g., cliffs and places where the ground curvature changes significantly. Also, I usually generate 25 cm contours using the raw lidar data and use these to judge where I can simplify.
